iiwa14_no_collision.sdf.in

```diff
@@ -0,0 +1,469 @@
+      <child>iiwa_link_1</child>
+      <parent>iiwa_link_0</parent>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>-2.96706</lower>
+          <upper>2.96706</upper>
+          <effort>300</effort>
+          <velocity>10</velocity>
+        </limit>
+        <dynamics>
+          <damping>0.0</damping>
+          <friction>0</friction>
+          <spring_reference>0</spring_reference>
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+      </axis>
+    </joint>
+    <link name="iiwa_link_2">
+      <pose relative_to="iiwa_joint_2"/>
+      <inertial>
+        <pose>0.0003 0.059 0.042 0 0 0</pose>
+        <mass>6.35</mass>
+        <inertia>
+          <ixx>0.0305</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0304</iyy>
+          <iyz>0</iyz>
+          <izz>0.011</izz>
+        </inertia>
+      </inertial>
+      <visual name="iiwa_link_2_visual_grey">
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>file://PWD_GOES_HERE/meshes/visual/link_2_grey.obj</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>0.4 0.4 0.4 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="iiwa_link_2_visual_orange">
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>file://PWD_GOES_HERE/meshes/visual/link_2_orange.obj</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>1.0 0.423529411765 0.0392156862745 1.0</diffuse>
+        </material>
+      </visual>
+      <gravity>1</gravity>
+      <velocity_decay/>
+      <self_collide>0</self_collide>
+    </link>
+    <joint name="iiwa_joint_2" type="revolute">
+      <pose relative_to="iiwa_link_1">
+        0 0 0.2025 1.570796326794897 0 3.141592653589793
+      </pose>
+      <child>iiwa_link_2</child>
+      <parent>iiwa_link_1</parent>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>-2.0944</lower>
+          <upper>2.0944</upper>
+          <effort>300</effort>
+          <velocity>10</velocity>
+        </limit>
+        <dynamics>
+          <damping>0.0</damping>
+          <friction>0</friction>
+          <spring_reference>0</spring_reference>
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+      </axis>
+    </joint>
+    <link name="iiwa_link_3">
+      <pose relative_to="iiwa_joint_3"/>
+      <inertial>
+        <pose>0 0.03 0.13 0 0 0</pose>
+        <mass>3.5</mass>
+        <inertia>
+          <ixx>0.025</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0238</iyy>
+          <iyz>0</iyz>
+          <izz>0.0076</izz>
+        </inertia>
+      </inertial>
+      <visual name="iiwa_link_3_visual">
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>file://PWD_GOES_HERE/meshes/visual/link_3.obj</uri>
```

Comment:
Is there some way to remove the need for template substituation?
Can we instead just hack ROS2's equivalent of a package path?

Reply:
Maybe it could use `package://` to a ROS package with [`drake::multibody::PackageMap`](https://github.com/RobotLocomotion/drake/blob/afd584c4b38ca168d12ed864a079309ee1eed88c/multibody/parsing/package_map.h) being made aware of that package.

Reply:
Yeah, it should already be aware.

FTR, we do want this to eventually be referred to `package://drake/...` so it won't step on the toes of other stuff.

Reply:
I think it makes sense to continue this conversation on the other thread: https://github.com/sloretz/drake_ros2_demos/pull/3#discussion_r516817838 starting with  68615ac